views/blocks/order_button.tpl

```diff
@@ -9,8 +9,51 @@
         </button>
         <div class="clearfix"></div>
     </div>
+[{elseif $payment->oxpayments__oxid->value == "wdpayolution-inv" && $payment->oxpayments__trusted_shop->value}]
+    <form action="[{$oViewConf->getSslSelfLink()}]" method="post" id="orderConfirmAgbBottom" class="form-horizontal">
+
+        <div class="hidden">
+            [{$oViewConf->getHiddenSid()}]
+            [{$oViewConf->getNavFormParams()}]
+            <input type="hidden" name="cl" value="order">
+            <input type="hidden" name="fnc" value="[{$oView->getExecuteFnc()}]">
+            <input type="hidden" name="challenge" value="[{$challenge}]">
+            <input type="hidden" name="sDeliveryAddressMD5" value="[{$oView->getDeliveryAddressMD5()}]">
+
+            [{if $oView->isActive('PsLogin') || !$oView->isConfirmAGBActive()}]
+                <input type="hidden" name="ord_agb" value="1">
+            [{else}]
+                <input type="hidden" name="ord_agb" value="0">
+            [{/if}]
+            <input type="hidden" name="oxdownloadableproductsagreement" value="0">
+            <input type="hidden" name="oxserviceproductsagreement" value="0">
+        </div>
+
+        <div class="well well-sm cart-buttons">
+            <div>
```

Comment:
If you use a `<label>` tag here, the text can also be clicked to tick the checkbox.

Reply:
Good tip 👍